css/src/tokens/features.scss

```diff
@@ -1 +1,5 @@
+/**
```

Comment:
@ollips Did we forget to clean this one up?

Reply:
Yes, that should be removed. If I remember it correctly, I added the feature flag before we've decided we're not going to move that way.

As for the docs - table styles are not connected there yet.

Reply:
Removed the file.